docs/setup.rst

```diff
@@ -80,122 +121,59 @@ Steps are below on getting your local development environment running:
-Before installing the python requirements, make sure to have PostgreSQL setup.
-
-**On mac**, `install Homebrew <https://brew.sh/>`_ and run:
-
-.. code-block:: bash
-
-    brew install postgres
-    pg_ctl -D /usr/local/var/postgres start && brew services start postgresql
-
-**On Ubuntu 18.x.x or Ubuntu 20.x.x** use apt-get:
-
-.. code-block:: bash
-
-    sudo apt-get update
-    sudo apt-get install postgresql python-psycopg2 libpq-dev libxslt-dev libxml2-dev
-
-**On CentOS / Fedora** use yum:
-
-.. code-block:: bash
-
-    sudo yum install postgresql gcc python-lxml postgresql-libs libxslt-devel libxml2-devel
-
-Install Python Requirements
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. note:: **ATTENTION MAC USERS:** you must install the xcode command line tools via ``xcode-select --install`` before proceeding.
-
-All python dependencies are kept in a file called ``requirements.txt``. Anytime a dependency is added or changed, we update it in this file. To bring your virtual environment up to date with all of these requirements easily, simply execute:
-
-.. code-block:: bash
-
-    pip3 install --upgrade pip
-    pip3 install -r requirements.txt
-
-Install Node Packages
-~~~~~~~~~~~~~~~~~~~~~~
-Node and node package manager are the backbone of our frontend setup. To begin, install Node Package Manager (npm).
-
-**On mac**:
-
-.. code-block:: bash
-
-    brew install node
-
-**On Ubuntu 18.x.x or Ubuntu 20.x.x**:
-
-.. code-block:: bash
-    
-    sudo apt-get install wget
-    wget -qO- https://deb.nodesource.com/setup_14.x | sudo bash -
-    sudo apt-get install -y nodejs
-
-**On CentOS / Fedora**:
-
-.. code-block:: bash
-
-    sudo yum install -y gcc-c++ make
-    curl -sL https://rpm.nodesource.com/setup_14.x | sudo -E bash -
-    sudo yum install nodejs
-
-Then use the newly installed Node Package Manager (npm) to install all javascript dependencies. When you execute this command, it reads from the file ``package.json`` which specifies all dependencies, their versions, and some additional node related configurations:
-
-.. code-block:: bash
-
-    sudo npm install
+    .. note:: If you are using WSL 2, you may, but not necessarily, need 
+    additional setup described in this `guide
+    <https://docs.docker.com/desktop/windows/wsl/>`_. If you run into 
+    additional errors, try the following:
+    1. Change "buildkit" from ``true`` to ``false`` in Settings -> Docker Engine. 
+    2. Refer to the Docker troubleshooting document `here
+    <https://github.com/microsoft/vscode-docker/wiki/Troubleshooting>`_
+
+    Open a browser and visit http://jhu.sem.ly:8000 to verify you have
+    Semester.ly running.
+
+Setting up Postgres
+~~~~~~~~~~~~~~~~~~~
+You can easily access the Postgres database within VSCode by following the next
+steps. You should have the Postgres extension by Chris Kolkman installed.
+
+1. Open the Postgres explorer on the left pane and click the plus button in the
+top right of the explorer to add a new database connection.
```

Comment:
Check your spacing here. You need to put newlines between each numbered element to make it appear correctly when generated into html/md format.